opensaml/2.6.4.wso2v2/pom.xml

```diff
@@ -0,0 +1,330 @@
+                        </Export-Package>
+                        <Import-Package>
+                            !org.opensaml.*,
+                            javax.crypto.*;version="${javax.crypto.version}",
+                            javax.net.*;version="${javax.net.version}",
+                            javax.security.*;version="${javax.security.version}",
+                            javax.servlet.jsp.*; version="${javax.servlet.jsp.version}",
+                            javax.servlet.*;version="${javax.servlet.version}",
+                            javax.xml.*;version="${javax.xml.version}",
+                            net.jcip.annotations.*;version="${net.jcip.annotations.version}";resolution:=optional,
+                            org.apache.commons.httpclient.*;version="${org.apache.commons.httpclient.version}";resolution:=optional,
+                            org.apache.commons.ssl.*;version="${org.apache.commons.ssl.version}";resolution:=optional,
+                            org.apache.commons.codec.binary;version="[1.4,2)",
+                            org.apache.velocity.*;resolution:=optional,
+                            org.apache.xml.security.*;version="${org.apache.xml.security.version}";resolution:=optional,
+                            org.bouncycastle.*;version="${org.bouncycastle.version}";resolution:=optional,
+                            org.joda.convert.*;version="${org.joda.convert.version}";resolution:=optional,
+                            org.owasp.esapi;resolution:=optional,
+                            org.slf4j.*;version="${org.slf4j.version}",
+                            org.w3c.*;version="${org.w3c.version}";resolution:=optional,
+                            org.xml.sax.*;version="${org.xml.sax.version}";resolution:=optional,
+                            junit.framework;resolution:=optional,
+                            junit.textui;resolution:=optional,
+                            org.apache.commons.codec.binary;resolution:=optional,
+                            org.apache.log4j;resolution:=optional,
+                            bsh;resolution:=optional,
+                            nu.xom;resolution:=optional,
+                            org.apache.commons.beanutils;resolution:=optional,
+                            org.apache.commons.collections.iterators;resolution:=optional,
+                            org.apache.commons.configuration;resolution:=optional,
+                            org.apache.commons.fileupload.*;resolution:=optional,
+                            org.apache.log4j.*;resolution:=optional,
+                            org.owasp.validator.html;resolution:=optional,
+                            org.apache.axiom.om.impl.dom.jaxp;version="${axiom.osgi.version.range}"
+                        </Import-Package>
+                        <Embed-Dependency>esapi;scope=compile|runtime;inline=false</Embed-Dependency>
```

Comment:
esapi is a library which supports output encoding and some additional security operations. In case we want to add this as a separate orbit we need to add an addition configuration class or configuration file for this to get work. opensaml library fully dependends on this library for encoding stuff to prevent XSS vulnerabilities and defines it's own configuration.